appveyor.yml

```diff
@@ -22,4 +22,13 @@ build_script:
 - ps: ./build.ps1
 test: off
 artifacts:
-- path: artifacts/packages/*.nupkg
\ No newline at end of file
+- path: artifacts/packages/*.nupkg
+deploy:
+- provider: NuGet
+  server: https://www.myget.org/F/skywalking/api/v2/package
+  api_key:
+    secure: y93uJrdiEYaeVTFxeCduQSlCussOUF4UztOyhkxyZxFnIco73J/K2rdA2QShoEuh
```

Comment:
Secure key is safe enough here?

Reply:
Sure. It is encrypted using [appveyor encrypt-tool](https://ci.appveyor.com/tools/encrypt)

Reply:
If it has been encrypted by CI platform, then good enough for me.